yarn.lock

```diff
@@ -481,6 +481,20 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@asap-hub/cms-migration@workspace:packages/cms-migration":
```

Comment:
adding a comment in here because I can't add a comment in `.pnp.cjs`. I don't know what is going on with my local .env but everytime I run `yarn clean cache && yarn install` this file `.pnp.cjs` get's deleted. Without deleting it when I run the script I get this error

```
 yarn workspace @asap-hub/cms-migration migrate:news
/Users/gabiayako/asap/asap-hub/.pnp.cjs:78169
    throw firstError;
    ^

Error: Your application tried to access yn, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.
```

Reply:
weird, is it just on your local?